x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/runtime_mappings/runtime_mappings.tsx

```diff
@@ -0,0 +1,243 @@
+      }
+    },
+    [cloneJob?.id]
+  );
+
+  return (
+    <>
+      <EuiSpacer size="s" />
+      <EuiFormRow
+        fullWidth={true}
+        label={i18n.translate('xpack.ml.dataframe.analytics.createWizard.runtimeMappingsLabel', {
+          defaultMessage: 'Runtime mappings',
+        })}
+      >
+        <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+          <EuiFlexItem grow={true}>
+            {runtimeMappings !== undefined && Object.keys(runtimeMappings).length > 0 ? (
+              <FormattedMessage
+                id="xpack.ml.dataframe.analytics.createWizard.runtimeMappingsListLabel"
+                defaultMessage="{runtimeFields}"
+                values={{
+                  runtimeFields: Object.keys(runtimeMappings).join(','),
+                }}
+              />
+            ) : (
+              <FormattedMessage
+                id="xpack.ml.dataframe.analytics.createWizard.noRuntimeMappingsLabel"
+                defaultMessage="No runtime mapping"
+              />
+            )}
+
+            {isRuntimeMappingsEditorEnabled && (
+              <>
+                <EuiSpacer size="s" />
+                <RuntimeMappingsEditor
+                  advancedEditorRuntimeMappingsLastApplied={
+                    advancedEditorRuntimeMappingsLastApplied
+                  }
+                  advancedRuntimeMappingsConfig={advancedRuntimeMappingsConfig}
+                  setIsRuntimeMappingsEditorApplyButtonEnabled={
+                    setIsRuntimeMappingsEditorApplyButtonEnabled
+                  }
+                  setAdvancedRuntimeMappingsConfig={setAdvancedRuntimeMappingsConfig}
+                  convertToJson={convertToJson}
+                  xJsonMode={xJsonMode}
+                />
+              </>
+            )}
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false} style={{ width: advancedEditorsSidebarWidth }}>
+            <EuiFlexGroup gutterSize="xs" direction="column" justifyContent="spaceBetween">
+              <EuiFlexItem grow={false}>
+                <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+                  <EuiFlexItem grow={false}>
+                    <EuiSwitch
+                      disabled={jobType === undefined}
+                      label={i18n.translate(
+                        'xpack.ml.dataframe.analytics.createWizard.advancedEditorRuntimeMappingsSwitchLabel',
+                        {
+                          defaultMessage: 'Edit runtime mappings',
+                        }
+                      )}
+                      checked={isRuntimeMappingsEditorEnabled}
+                      onChange={() => toggleEditorHandler()}
+                      data-test-subj="mlDataFrameAnalyticsRuntimeMappingsEditorSwitch"
+                    />
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false}>
+                    <EuiCopy beforeMessage={COPY_TO_CLIPBOARD_RUNTIME_MAPPINGS} textToCopy={''}>
```

Comment:
This button doesn't copy anything for me. Same thing on the Transforms wizard too actually. No text is copied to my clipboard.

Reply:
Could you copy over the fix used in https://github.com/elastic/kibana/pull/95996 here?

Reply:
To be fixed in a follow up 👍

Reply:
Fix added in c224c8cf620e4613ef0a39ded34950f348bb9426